libraries/CurieTime/src/CurieTime.cpp

```diff
@@ -145,3 +128,25 @@ void setTime(int hour, int minute, int second, int day, int month, int year)
 
     setTime(t);
 }
+
+/*
+ * CurieTime.cpp - Time library for Arduino101
+  
```

Comment:
There is trailing whitespace on several lines here (although github does not hilight it)